apiserver/application/application_unit_test.go

```diff
@@ -29,7 +30,7 @@ type ApplicationSuite struct {
 	endpoints []state.Endpoint
 	relation  mockRelation
 
-	env          *mockEnviron
+	env          environs.Environ
```

Comment:
Doesn't doing this make the places where you need to set up mock environ data more awkward?